### DISCUSSION TITLE
Hilangkan "Date" Caption pada Image

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#157] [Roadmap]
Berdasarkan thread #156 pesan bertime `image` otomatis akan memiliki caption tanggal hari ini jika tidak diisi.

### Expected

Caption pada image tetap kosong jika memang tidak diisi.

**Comment by adidesai18** [comment]
Even image preview is not correct.

**Reply by rioastamal** [reply]
@adidesai18 as I said on #156, the preview image (white question mark with green background) is intended and I have no plan to change it.

**Comment by rioastamal** [comment]
Fitur sudah diimplementasikan pada v1.0-RC20. Lihat #20

**Comment by adidesai18** [comment]
Can I get to know the reason behind this?

**Reply by rioastamal** [reply]
Saving compute resource. Generating thumbnail means downloading and modifying original image. This is currently the only reason. It may change in the future when I introduce paid package, but for now I am not prioritizing it.

**Reply by anargyaid** [reply]
atau at least jangan tanda tanya ijo mas? abu-abu aja kaya skeleton loader, client saya ngiranya virus :'(

**Comment by adidesai18** [comment]
Any idea when is this paid version is launching

**Comment by adidesai18** [comment]
When is paid version is launching ?
